cmd/app.go

```diff
@@ -326,3 +352,41 @@ func lastReadJournalTimeStamp() uint64 {
 
 	return lastRead
 }
+
+func handleAuditLogMetrics(
+	ctx context.Context,
+	eg *errgroup.Group,
+	pprov *metrics.PrometheusMetricsProvider,
+	auditMetricsSecondsInterval int,
+	auditLogWriteTimeSecondThreshold int,
+	enableAuditMetrics bool,
+) {
+	if !enableAuditMetrics {
+		return
+	}
+
+	auditLogFilePath := "/var/log/audit/audit.log"
+
+	eg.Go(func() error {
+		tickChan := time.NewTicker(time.Second * time.Duration(auditMetricsSecondsInterval)).C
```

Comment:
You should call `defer Ticker.Stop()` after creating the Ticker.

Reply:
.C just gives me the channel. dont have the ticker at this point.